rasa/shared/data.py

```diff
@@ -69,6 +70,10 @@ def get_test_directory(paths: Optional[Union[Text, List[Text]]]) -> Text:
     Returns:
         Path to temporary directory containing all found Core training files.
     """
+    rasa.shared.utils.io.raise_warning(
```

Comment:
can you use `raise_deprecation_warning`?

Reply:
changed these back as I need type `DeprecationWarning` in this case and I can't specify a category with `raise_deprecation_warning`

Reply:
why do you specifically need `DeprecationWarning`? What's the requirement from the user perspective?

Reply:
I need a deprecation warning because I want to warn other developers or users using the Python API. This isn't called as part of anything when using the CLI (this would make a `FutureWarning` necessary). **If** anybody decides to use that in their Python code then they'll see that it's deprecated (e.g. Pycharm automatically strikes it through)